impl/src/test/java/org/ehcache/docs/Tiering.java

```diff
@@ -35,6 +51,18 @@
   public TemporaryFolder folder = new TemporaryFolder();
 
   @Test
+  public void tierSizing() {
+    // tag::heap[]
+    ResourcePoolsBuilder.newResourcePoolsBuilder().heap(10, EntryUnit.ENTRIES); // <1>
+    ResourcePoolsBuilder.newResourcePoolsBuilder().heap(10); // <2>
```

Comment:
I don't get it. The 3 lines of code are of course not valid. They are there to show the 3 ways to create a heap tier. I guess that wasn't clear in the documentation. Let me try something